builder/vmware/common/step_upload_tools.go

```diff
@@ -33,7 +33,7 @@ func (c *StepUploadTools) Run(ctx context.Context, state multistep.StateBag) mul
 
 	if c.RemoteType == "esx5" {
 		if err := driver.ToolsInstall(); err != nil {
-			state.Put("error", fmt.Errorf("Couldn't mount VMware tools ISO. Please check the 'guest_os_type' in your template.json."))
+			state.Put("error", fmt.Errorf("unable to mount vmware tools iso, check the 'guest_os_type'"))
```

Comment:
Same capitalisation problem I think here with ISO

Reply:
Resolved in 67054cd.